pom.xml

```diff
@@ -90,17 +90,46 @@
                     <mapping>
                         <xq>xquery_style</xq>
                         <xqm>xquery_style</xqm>
+                        <xql>xquery_style</xql>
+                        <xconf>xml_style</xconf>
                     </mapping>
                     <excludes>
                         <exclude>pom.xml</exclude>
                         <exclude>README.md</exclude>
                         <exclude>LICENSE</exclude>
                         <exclude>xquery-license-style.xml</exclude>
                         <exclude>xar-assembly.xml</exclude>
+                        <exclude>**/*.ent</exclude>
                     </excludes>
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>xml-maven-plugin</artifactId>
+                <version>1.0.1</version>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>validate</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <validationSets>
+                        <validationSet>
+                            <dir>src/main/xar-resources/data</dir>
+                            <excludes>
+                                <exclude>*.xq</exclude>
+                                <exclide>xinclude.xml</exclide>
```

Comment:
Typo alert! `<exclide>` should be `<exclude>`.

Reply:
Thanks! I have fixed that now.